neard/src/lib.rs

```diff
@@ -48,11 +57,30 @@ pub fn get_default_home() -> String {
     }
 }
 
+/// Function checks current version of the database and applies migrations to the database.
+pub fn apply_store_migrations(path: &String) {
```

Comment:
Should this take another function as argument? Or maybe make a trait out of it?

Reply:
why? just add calling to migrations inside the function.